ucp/public_api.py

```diff
@@ -24,6 +25,34 @@ def _get_ctx():
 # the functions here.
 
 
+def init(options={}, env_takes_precedence=False):
```

Comment:
Can we also be allowed to pass the `UCXPY_IFNAME` here? It seems that this isn't possible now.

Reply:
That would also need to appear in the `get_config()` then.

Reply:
Nevermind, @quasiben helped me realize `UCXPY_IFNAME` isn't necessary to use with Dask.